include/GafferImage/ImageWriter.h

```diff
@@ -37,6 +37,8 @@
 #ifndef GAFFERIMAGE_IMAGEWRITER_H
 #define GAFFERIMAGE_IMAGEWRITER_H
 
+#include "Gaffer/Expression.h"
```

Comment:
To keep the headers as clean as we can, let's use a forward declaration (in the section starting at line 46 below) rather than include Expression.h here.